scripts/tests/scripts_regression_tests.py

```diff
@@ -22,6 +22,10 @@
 from  CIME.code_checker import check_code, get_all_checkable_files
 from  CIME.test_status import *
 
+from  CIME.case_setup import case_setup
+from  CIME.case_submit import submit
+import CIME.build as build
```

Comment:
Did you run pylint on this?  It seems you are not using submit or build and can remove them.

Reply:
I knew I was forgetting something. Fixed, and now passes pylint